.eslintrc

```diff
@@ -31,9 +31,23 @@
     //   exports is not enabled yet because that requires setting up type-aware
     //   linting.
     "@typescript-eslint/consistent-type-assertions": "error",
-    "@typescript-eslint/consistent-type-imports": "error"
+    "@typescript-eslint/consistent-type-imports": "error",
   },
   "overrides": [
+    // Additional rules that require type information, and thus can only
+    // run on a subset of the linted code.
```

Comment:
"subset" here means only those files which are listed as included in `tsconfig.json`. This excludes tests, the UI playground and some build scripts that run in Node.

Reply:
Could you amend the comment such that it clarifies how the _subset_ is computed? It seems that it might be a combination of `tsconfig.json` config and the `files` and `excludedFiles` fields here?